Linear_Regression/LinearRegression Exercises.ipynb

```diff
@@ -0,0 +1,1442 @@
+{
```

Comment:
<p>That would actually be good to "trap" them and not remove the missing data. They'll see what a failing model looks like and how to fix it ;)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Added above, I love the idea but the model is not failing unfortunately, PyMC imputes the values and the students might miss the point</p>

Reply:
<p>Really? So there are NaNs for some observations and PyMC imputes them without any masked array stuff??</p><p> </p>

Reply:
Yep it seems like it